backend/composer/models.py

```diff
@@ -716,6 +717,9 @@ def save(self, *args, **kwargs):
             self.reference_uri = create_reference_uri(self.pk)
             self.save(update_fields=["reference_uri"])
 
+        # recompute - journey
+        self.journey_path = compile_journey(self)
+        super().save(update_fields=["journey_path"])
```

Comment:
Question: Do we need to update the journey on connectivity statement changes? 
In other words, what change affects the journey that is not capture in the signals?

Computing the journey is somewhat expensive so I would prefer that we make sure we only do it when we really need to

Reply:
Interesting point @afonsobspinto, I agree - so we just do the computation of the journey path only in the signals then. When the entities are changed, and therefore the journey should also be changed.  

Signals should cover the requirements for us. thanks for pointing this!
Made the change.